packages/cozy-stack-client/src/CozyStackClient.js

```diff
@@ -65,8 +65,9 @@ class CozyStackClient {
    */
   async fetch(method, path, body, opts = {}) {
     const options = { ...opts }
+    const headers = { ...(opts.headers || {}) }
```

Comment:
Nit: spreading `undefined` works, so doing

```js
const headers = { ...opt.headers }
```

works even if `opt.headers` is undefined, in which case `headers` will be an empty object:

```
const opts = {}
undefined
const headers = { ...opts.headers }
undefined
headers
Object {  }
```